innovation_sweet_spots/utils/read_from_s3.py

```diff
@@ -0,0 +1,39 @@
+"""
+innovation_sweet_spots.utils.nihr.read_from_s3
+
+Functions for saving and loading data into the S3 buckey
+"""
+import json
+import boto3
+import io
+import pandas as pd
+from pathlib import Path
+from functools import lru_cache
+
+BUCKET_NAME = "innovation-sweet-spots-lake" # put in .config
+
+def save_to_s3(s3_path, filename, contents):
```

Comment:
Hey, could you please indicate the types of the input and output variables? (you can check out other getters to see how we're trying to do it). It's really helpful to get a quick idea about the input and output types.

More info on typing here: https://docs.python.org/3/library/typing.html